sbsp_frontend/src/api/websocket.ts

```
@@ -318,9 +347,39 @@ export function useWebsocketApi(): IBackendAdapter {
               websocketApiState.fullStateResolver[0](msg.data);
             }
             websocketApiState.projectStatus = msg.data.projectStatus;
+            break;
+          case 'error':
+            let error: BackendError = {
+              type: 'custom',
+              id: 0,
+              message: 'Unknown error occured.',
+            };
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)